integration/environment/quarks_job_cmd.go

```diff
@@ -37,6 +38,8 @@ func (q *QuarksJobCmd) Start(namespace string) error {
 		"-a", namespace,
 		"-o", "cfcontainerization",
 		"-r", "quarks-job",
+		"--meltdown-duration", strconv.Itoa(defaultTestMeltdownDuration),
```

Comment:
Without this integration storage tests are actually failing with a bigger meltdown window..